pkg/download/mode/mode_test.go

```diff
@@ -127,12 +129,12 @@ func TestNewFile_err(t *testing.T) {
 		{
 			name:     "empty mode",
 			mode:     "",
-			expected: downloadModeErr,
+			expected: Mode("").Validate(op).Error(),
```

Comment:
hmm, we probably shouldn't do string matching here: either 
1. change `expected string` to `hasErr bool` 
2. or, change `expected string` to `expected errors.Kind`